esdb/operations.go

```diff
@@ -0,0 +1,16 @@
+package esdb
+
+type ServerVersion struct {
+	Major int
+	Minor int
+	Patch int
+}
+
+func (client *Client) GetServerVersion() (ServerVersion, error) {
+	handle, err := client.grpcClient.getConnectionHandle()
+	if err != nil {
+		return ServerVersion{}, err
```

Comment:
I prefer we return `nil` when the `ServerVersion` is unavailable.

Reply:
Yeah just change the return type to `(*ServerVersion, error)`

Reply:
ahh yup thanks :man_facepalming:

Reply:
fixed.